packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -7,7 +7,10 @@ create table sign_in_experiences (
   language_info jsonb /* @use LanguageInfo */ not null,
   terms_of_use jsonb /* @use TermsOfUse */ not null,
   sign_in_methods jsonb /* @use SignInMethods */ not null,
```

Comment:
Sie v2 depends on `sign_in` and `sign_up` but not `sign_in_methods` in the future?
Are we going to make sie v1 compatible? Or will we drop `sign_in_methods` in another PR

Reply:
can not drop `sign_in_methods` until all jobs are done.